src/index.js

```diff
@@ -0,0 +1,65 @@
+import './style.css';
+// import bootstrap from "bootstrap";
+
+const toDoInput = document.querySelector('#toDoInput');
+const toDoList = document.querySelector('.toDoList');
+const form = document.querySelector('.form');
+// storage
+const addTodStorage = (arrToDos) => {
+  const storage = localStorage.setItem('ToDo', JSON.stringify(arrToDos));
+  return storage;
+};
+
+const getStorage = () => {
+  const storage = localStorage.getItem('ToDo') === null
+    ? []
+    : JSON.parse(localStorage.getItem('ToDo'));
+  return storage;
+};
+
+// array
+const arrToDos = getStorage();
```

Comment:
- [ ] So, At that point you have left one required change. 

> Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
description [string].
completed [bool].
index: [number].

Kindly follow the insturction and create your dinamic array. Also, make sure to display them when the page loaded. Thank you.

Reply:
@Rustamxon7
yes i did it

Reply:
class ToDos {
  constructor(index = 0, description, completed = false) {
    this.index = index;
    this.description = description;
    this.completed = completed;
  }
}

Reply:
const ToDoItem = new ToDos(id, toDoInput.value, false);

Reply:
arrToDos.push(ToDoItem);

Reply:
@Rustamxon7
please can you check it